### DISCUSSION TITLE
longan nano problem

### DISCUSSION THREAD

**Discussion by vorosj** [hathach/tinyusb#1992] [General]
Hi,
I tried to compile the uac2_headset example for the gd32vf103 longan nano board.
I got error message, what is the problem? Here is a copy of my wsl terminal:

`mmormota@DESKTOP-S2K747J:/mnt/c/tinyusb$ cd examples
mmormota@DESKTOP-S2K747J:/mnt/c/tinyusb/examples$ cd device
mmormota@DESKTOP-S2K747J:/mnt/c/tinyusb/examples/device$ cd uac2_headset
mmormota@DESKTOP-S2K747J:/mnt/c/tinyusb/examples/device/uac2_headset$ make BOARD=sipeed_longan_nano get-deps
python3 /mnt/c/tinyusb/tools/get_deps.py hw/mcu/gd/nuclei-sdk
cloning lib/FreeRTOS-Kernel with https://github.com/FreeRTOS/FreeRTOS-Kernel.git
cloning lib/lwip with https://github.com/lwip-tcpip/lwip.git
cloning tools/uf2 with https://github.com/microsoft/uf2.git
cloning hw/mcu/gd/nuclei-sdk with https://github.com/Nuclei-Software/nuclei-sdk.git
remote: Enumerating objects: 582, done.
remote: Counting objects: 100% (582/582), done.
remote: Enumerating objects: 23, done.
remote: Counting objects: 100% (23/23), done.
remote: Compressing objects: 100% (19/19), done.
remote: Enumerating objects: 437, done.
remote: Counting objects: 100% (437/437), done.
remote: Total 23 (delta 1), reused 14 (delta 1), pack-reused 0
remote: Compressing objects: 100% (381/381), done.
Unpacking objects: 100% (23/23), 28.12 KiB | 113.00 KiB/s, done.
remote: Enumerating objects: 772, done.
remote: Counting objects: 100% (772/772), done.
From https://github.com/microsoft/uf25/419)
 * branch            19615407727073e36d81bf239c52108ba92e7660 -> FETCH_HEAD
remote: Compressing objects: 100% (419/419), done.
remote: Compressing objects: 100% (513/513), done.
remote: Total 772 (delta 337), reused 579 (delta 318), pack-reused 0
Receiving objects: 100% (772/772), 1.21 MiB | 5.52 MiB/s, done.
Resolving deltas: 100% (337/337), done. MiB | 2.01 MiB/s
remote: Total 437 (delta 78), reused 148 (delta 35), pack-reused 0
Receiving objects: 100% (437/437), 1.28 MiB | 2.04 MiB/s, done.
Resolving deltas: 100% (78/78), done.
From https://github.com/FreeRTOS/FreeRTOS-Kernel
 * branch            def7d2df2b0506d3d249334974f51e427c17a41c -> FETCH_HEAD
From https://github.com/lwip-tcpip/lwip
 * branch            159e31b689577dbf69cf0683bbaffbd71fa5ee10 -> FETCH_HEAD
remote: Total 582 (delta 102), reused 274 (delta 45), pack-reused 0
Receiving objects: 100% (582/582), 11.56 MiB | 9.69 MiB/s, done.
Resolving deltas: 100% (102/102), done.
From https://github.com/Nuclei-Software/nuclei-sdk
 * branch            7eb7bfa9ea4fbeacfafe1d5f77d5a0e6ed3922e7 -> FETCH_HEAD
mmormota@DESKTOP-S2K747J:/mnt/c/tinyusb/examples/device/uac2_headset$ make BOARD=sipeed_longan_nano all
AS startup_gd32vf103_asm.o
make: riscv-none-embed-gcc: Command not found
make: *** [../../rules.mk:144: _build/sipeed_longan_nano/obj/hw/mcu/gd/nuclei-sdk/SoC/gd32vf103/Common/Source/GCC/startup_gd32vf103_asm.o] Error 127
mmormota@DESKTOP-S2K747J:/mnt/c/tinyusb/examples/device/uac2_headset$`

**Comment by hathach** [comment]
try to install suitable risv toolchain and use correct one, checkout family.mk to make needed changes
